docs/eog/eog-in-practice/cw23/feasibility-stage/fs-date.rst

```diff
@@ -0,0 +1,15 @@
+.. _cw23-fs-date:
+
+CW23 Date
+===========
+
+The date needs to fit into the calendar of other events that are likely to affect the :ref:`cw23-fs-audience`.
+Things to consider here may include being outside of University term time and preferably not clashing with school holidays for those with caring responsibilities.
+Bank holidays and religious holidays should also inform the decision.
+
+We wanted to avoid holding the event during Ramadan (which took place from 22 March -  21 April 2023), which meant that the event would have to take place during term time.
+The earliest we could hold a Tuesday - Thursday event was therefore the week of 24 April, however the EGU General Assembly 2023 took place from 23–28 April 2023, which we also wanted to avoid clashing with based on feedback from CW22.
+We note that there were a number of bank holidays during this period that also made scheduling difficult (1 May 2023 and 8 May 2023).
+
+However, we decided on the dates of Tuesday, 2 May to Thursday, 4 May 2023 for CW23.
+We could not schedule it later than this due to other Institute activities and end of financial year preparation by the Finance Team.
```

Comment:
`We could not schedule it later than this due to other Institute activities and end of financial year preparation by the SSI Finance Team.`